### ISSUE TITLE
Multi-arch for operators

### ISSUE THREAD

**Issue by Maleware** [stackabletech/operator-templating#165]
Changing operator templates to enable multi arch builds.

Steps: 
1. docker buildx as standard build option
2. organization as a argument

**Comment by sbernauer** [comment]
Needs ADR from https://github.com/stackabletech/documentation/pull/259

**Comment by lfrancke** [comment]
This is "In Progress" for Refinement. Is that still true?

**Comment by Maleware** [comment]
No, wanted to look over it this morning. I think it's good now. Should I move it to waiting for?

**Comment by lfrancke** [comment]
When you're done with Refining you can move it to the right, yes. "Refinement Acceptance: Waiting for"

**Comment by Maleware** [comment]
Ok, done.

**Comment by lfrancke** [comment]
I'll transfer this issue to the issues repository
